xml_inscriptions/transcribed/MA.Wor.WAM.L.1915.75.xml

```diff
@@ -0,0 +1,205 @@
+                    </bibl>
+                    <bibl>
+                        <ref type="external" target="http://www.edr-edr.it/edr_programmi/res_complex_comune.php?do=book&amp;id_nr=EDR143228&amp;fo_antik=&amp;fo_modern=&amp;Bibliografia%5B%5D=&amp;Testo=&amp;boolTesto=AND&amp;Testo2=&amp;bool=AND&amp;ordinamento=id_nr&amp;javasi=javascriptsi&amp;se_foto=tutte&amp;lang=en">EDR143228</ref>
+                    </bibl>
+                </listBibl>
+            </sourceDesc>
+        </fileDesc>
+
+        <!--  *************************************  <encodingDesc>  ********************************* -->
+
+        <encodingDesc>
+
+                <xi:include
+                    href="http://library.brown.edu/usep_data/resources/include_taxonomies.xml">
+                    <xi:fallback>
+                        <p>Taxonomies for US Epigraphy controlled values</p>
+                    </xi:fallback>
+                </xi:include>
+
+        </encodingDesc>
+
+        <profileDesc> <!-- not used --> </profileDesc>
+
+
+        <!--  *************************************  <revisionDesc>  ********************************* -->
+    	<!-- Record all changes here -->
+
+        <revisionDesc>
+            <change when="2023-12-14" who="Advaith Rege">Initial Encoding</change>
+        </revisionDesc>
+
+    </teiHeader>
+
+    <!--  *************************************  <facsimile>  *********************************  -->
+        <!-- Used to record images of an inscription.   -->
+
+    <facsimile>
+        <surface>
+            <desc/>
+            <graphic url="MA.Wor.WAM.L.1915.75-LeftSide-sml.jpg"/>
+        </surface>
+        <surface>
+            <desc>detail of inscription</desc>
+            <graphic url="MA.Wor.WAM.L.1915.75-sml.jpg"/>
+        </surface>
+    </facsimile>
+
+    <text>
+        <body>
+            <div type="edition">
+            <ab>
+                <lb n="1"/><expan><abbr>D</abbr><ex>is</ex></expan> <expan><abbr>M</abbr><ex>anibus</ex></expan> <expan><abbr>S</abbr><ex>acrum</ex></expan>
+                <lb n="2"/>
+                    <persName>
+                        <name key="Nicanor">Nicanori</name>
+                    </persName>
+                    <expan><abbr>fec</abbr><ex>erunt</ex></expan>
+                <lb n="3"/>
+                    <persName>
+                        <name type="gentilicium" key="Cincia">Cincia</name>
+                        <name type="cognomen" key="Trophime">Trophime</name>
+                    </persName>
+                <lb n="4"/>et 
+                    <persName>
+                        <name key="Hermes">Hermes</name>
+                    </persName>
+                    <expan><abbr>cons</abbr><ex>ervo</ex></expan>
+                <lb n="5"/>
+                    <expan><abbr>B</abbr><ex>ene</ex></expan> <expan><abbr>M</abbr><ex>erenti</ex></expan>.
```

Comment:
b and m should be lower case; I think we understand them in apposition to "Nicanori... conservo"